### PR TITLE
Popup Stoyanova

### PR DESCRIPTION
**Pull request by AnnaStoyano**
Please, review)
[Demo](https://annastoyano.github.io/PopUp/index)
[Code](https://github.com/AnnaStoyano/PopUp)

### CODE REVIEW

**Comment by madmaxWMFU** [comment]
You forgot to insert links to the demo version and source code

**Comment by AnnaStoyano** [comment]
I don`t understand how I should create demo. Should I create new repo with this popup folder and make GitHub page?

**Comment by madmaxWMFU** [comment]
Yes that's right
Just create a repository for this task make [GitHub pages](https://docs.github.com/en/free-pro-team@latest/github/working-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site)
And then edit the description by adding two links

**Comment by AnnaStoyano** [comment]
Thank you)

**Comment by AnnaStoyano** [comment]
I've changed description) Please, can you review again?

File: submissions/annStoyano/html-css-popup/index.html [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+</head>
+
+<body>
+    <header class="header">
+        <nav class="navigation">
+            <a href="#" class="mails link">Mails</a>
+            <a href="#" class="mails link">Images</a>
+            <section class="popup-wrapper">
+                <input type="checkbox" id='popup-check' class='popup_btn'>
+                <label class="popup_btn-icon" for='popup-check'></label>
```
Comment (madmaxWMFU) [inline submissions/annStoyano/html-css-popup/index.html:18]: Navigation is best used in this format, the sample from [MDN](https://developer.mozilla.org/uk/docs/Web/HTML/%D0%95%D0%BB%D0%B5%D0%BC%D0%B5%D0%BD%D1%82/nav)
```
<nav>
  <ul>
    <li><a href="#">Home</a></li>
    <li><a href="#">About</a></li>
    <li><a href="#">Contact</a></li>
  </ul>
</nav>
```

File: submissions/annStoyano/html-css-popup/index.html [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+            <a href="https://github.com/AnnaStoyano" class="user">
+                <figure class="user-photo">
+                    <img src="popup-icons/favicon.png" alt="kottans">
+                </figure>
+            </a>
+        </nav>
+    </header>
+</body>
+
+</html>
```
Comment (madmaxWMFU) [inline submissions/annStoyano/html-css-popup/index.html:155]: Add an empty new line. 
You should always leave a blank line at the end of the file.

**Review by madmaxWMFU — CHANGES_REQUESTED** [review]
Good job!
Please, see the comments below.

**Review by madmaxWMFU — APPROVED** [review]

File: submissions/annStoyano/html-css-popup/index.html [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+            <ul class="nav-items">
+                <li><a href="#" class="mails link">Mails</a></li>
+                <li><a href="#" class="mails link">Images</a></li>
+                <li class="popup-wrapper">
+                    <input type="checkbox" id='popup-check' class='popup_btn'>
+                    <label class="popup_btn-icon" for='popup-check'></label>
+                    <div class="popup">
+                        <div class="popup-content">
+                            <section class="main-menu">
+                                <a href="#" class="popup-item">
```
Comment (madmaxWMFU) [inline submissions/annStoyano/html-css-popup/index.html:23]: Maybe someone of mentors will advise you here to use this tag (`section`) correctly
I would use the usual `div`

File: submissions/annStoyano/html-css-popup/scss/style.scss [hunk 4]
```diff
@@ -0,0 +1,263 @@
...
+    align-content: space-between;
+    flex-wrap: wrap;
+    padding: 40px 20px 10px 20px;
+    width: 100%;
+    height: 55%;
+}
+
+.kottans{
+    margin: 10px auto 0px auto;
+}
```
Comment (madmaxWMFU) [inline submissions/annStoyano/html-css-popup/scss/style.scss:263]: Don't forget about a new line.

You can also set your text editor for adding a new empty line

**Comment by madmaxWMFU** [comment]
As for me, you are done with this task.
Wait for changes from mentors.

**Review by MaxKalashnyk — COMMENTED** [review]
Good job! Approved!

**Review by MaxKalashnyk — APPROVED** [review]

**Comment by madmaxWMFU** [comment]
Maybe make a merge?

**Comment by madmaxWMFU** [comment]
Сorrect mistakes from [HTML validator](https://validator.w3.org/nu/?doc=https%3A%2F%2Fnazar-cmd.github.io%2Fpopup%2F) and [CSS validator](https://jigsaw.w3.org/css-validator/validator?uri=https%3A%2F%2Fnazar-cmd.github.io%2Fpopup%2F&profile=css3svg&usermedium=all&warning=1&vextwarning=&lang=ru)

**Pull request merged**